api-catalog-ui/frontend/package.json

```diff
@@ -36,13 +36,13 @@
         "redux-thunk": "2.3.0",
         "rimraf": "3.0.2",
         "rxjs": "6.3.3",
-        "swagger-ui": "3.25.0",
+        "swagger-ui-react": "^3.23.0",
         "uuid": "3.3.2"
     },
     "scripts": {
         "start": "cross-env HTTPS=true react-scripts start",
         "start:dev": "concurrently -c blue,green -n Backend,UI \"npm run start:mocked-backend\" \"cross-env REACT_APP_GATEWAY_URL=http://localhost:8000 npm run start\"",
-        "start:mocked-backend": "cross-env REACT_APP_GATEWAY_URL=http://localhost:8000 nodemon mocked-backend/server.js",
+        "start:mock": "cross-env REACT_APP_GATEWAY_URL=http://localhost:8000 nodemon mocked-backend/server.js",
```

Comment:
It was a change coming from Lena branch. Please either update the documentation with the new correct name `start:mock` or revert it to `start:mocked-backend`.